nbs/03_process.ipynb

```diff
@@ -32,11 +32,16 @@
     "from nbdev.maker import *\n",
```

Comment:
<p>Markdown is best used for user-facing docs of what a class does, rather than how it works. Comments are generally best for documenting how it works - ideally at the site it occurs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/928/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>